backend/app/tests/api/routes/test_threads.py

```
@@ -14,7 +14,7 @@
     handle_openai_error,
     poll_run_and_prepare_response,
 )
-from app.models import APIKey, OpenAI_Thread
+from app.models import APIKey, OpenAI_Thread, APIKeyPublic
```

Ruff (0.12.2)

`app.models.APIKey` imported but unused

Remove unused import

(F401)